SummerProject/article/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+from user.models import User
+from category_team.models import Team, Category, SocialNetworkChoice
+
+
+class Article(models.Model):
+    headline = models.CharField(max_length=150, blank=False)
```

Comment:
FYI: the default value of `blank` is `False`, so passing `blank=False` is redundant

Reply:
Done